addons/mass_mailing/models/mailing.py

```diff
@@ -741,6 +755,11 @@ def _action_view_documents_filtered(self, view_filter):
             helper_header = _("No %s address bounced yet!", model_name)
             helper_message = _("Bounce happens when a mailing cannot be delivered (fake address, "
                                "server issues, ...). Check each record to see what went wrong.")
+        elif view_filter == 'clicked':
+            found_traces = self.mailing_trace_ids.filtered(lambda trace: trace.links_click_ids)
+            helper_header = _("No %s clicked your mailing yet!", model_name)
+            helper_message = _(
+                "Come back once your mailing has been sent to track who clicked on the link you sent.")
```

Comment:
That's twice "sent", not super elegant.

Maybe "Come back once your mailing has been sent to track who clicked on the embedded links." ?